### ISSUE TITLE
VRAM of GPUs for training and reasoning

### ISSUE THREAD

**Issue by Jiatttliu** [horizon-research/Fov-3DGS#1]
Hello! I noticed that you completed the training on the `Nvidia Jetson Xavier` board, and I would like to ask you how much VRAM is used in the training and reasoning process?
waiting for your reply!

**Comment by linwk20** [comment]
We performed training on an RTX 4090 GPU and used a Jetson for inference rendering. 
The 4090's 24GB memory is sufficient for all scenes tested during training. 
For inference on Jetson, we needed to disable data caching on GPU memory to prevent out-of-memory (OOM) errors. We achieved this by using fps_mode as shown in [this code snippet](https://github.com/horizon-research/Fov-3DGS/blob/4f4777de83832dbe4aab614b237506c7fd2d82a4/fov3dgs/scene/__init__.py#L25). After disabling data caching, we successfully ran all scenes on Jetson. It generally takes less than 5GB memory if i remember correctly.

**Comment by Jiatttliu** [comment]
Thanks for your reply! 5GB is very friendly for most embedded development boards, It's really a commendable job!



At 2024-07-11 05:14:04, "linwk20" ***@***.***> wrote:

We performed training on an RTX 4090 GPU and used a Jetson for inference rendering.
The 4090's 24GB memory is sufficient for all scenes tested during training.
For inference on Jetson, we needed to disable data caching on GPU memory to prevent out-of-memory (OOM) errors. We achieved this by using fps_mode as shown in this code snippet. After disabling data caching, we successfully ran all scenes on Jetson. It generally takes less than 5GB memory if i remember correctly.
